db/migrate/20150720152332_attach_existing_lists_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AttachExistingListsToUser < ActiveRecord::Migration
+  def up
+    user = User.find_by_email 'vasyll@interlink-ua.com'
+    lists = List.all
+
+    lists.update_all user_id: user.id
```

Comment:
@vlasiak, Can you please try to do it without retrieving all lists into the memory?
```ruby
user = User.find_by_email 'vasyll@interlink-ua.com'
List.update_all user_id: user.id
``` 
Currently there is no problem to fetch all records, but this is about to learn the best practice of migration all existing data.